circup/backends.py

```diff
@@ -109,9 +115,16 @@ def install_module(
                            with get_bundle_versions()
         :param bool upgrade: Upgrade the specified modules if they're already installed.
         """
+        local_path = None
         if not name:
             click.echo("No module name(s) provided.")
-        elif name in mod_names:
+        elif name in mod_names or os.path.exists(name):
+            if os.path.exists(name):
+                # local file exists use that.
+                local_path = name
+                name = local_path.split(os.path.sep)[-1]
+                name = name.replace(".py", "").replace(".mpy", "")
+
```

Comment:
I think it would be good to print out something noting that you're installing from a local file/directoyr instead of the bundle. The names might be the same and that might be really confusing if you're accidentally shadowing a library from the bundle.

If you restructure this slightly you can avoid calling `os.path.exists(name)` twice. I think you can return after the `click.echo()`, and then change the `elif` to an `if`.

Reply:
Thank you. With the latest commit I've refactored this section to return early in the no module case and only checking path exists once.